integration-tests/src/test/java/oracle/weblogic/kubernetes/ItElasticLogging.java

```diff
@@ -343,6 +366,69 @@ void testWlsLoggingExporter() throws Exception {
     logger.info("Query WebLogic log info succeeded");
   }
 
+  /**
+   * Test when new variable createLogStashConfigMap sets to true, a configMap named weblogic-operator-logstash-cm
+   * is created and users can update logstash configuration by updating the configmap
+   * instead of rebuilding operator image.
+   */
+  @Test
+  @DisplayName("Test that configMap weblogic-operator-logstash-cm is created and able to be modified")
+  void testCreateLogStashConfigMapModify() throws Exception {
+    String configMapName = "weblogic-operator-logstash-cm";
+    List<Path> logstashConfigFiles = new ArrayList<>();
+
+    // verify that configMap weblogic-operator-logstash-cm is created when createLogStashConfigMap= = true
+    Callable<Boolean> configMapExist = assertDoesNotThrow(() -> configMapExist(opNamespace, configMapName),
+        "copy logstash.conf failed");
+    assertTrue(configMapExist.call().booleanValue(),
+        String.format("configMap %s in namespace %s DO NOT exist", configMapName, opNamespace));
+
+    // copy the logstash config file to workdir
+    assertDoesNotThrow(() -> copy(Paths.get(sourceConfigFile), Paths.get(destConfigFile)),
+        "copy logstash.conf failed");
+    logstashConfigFiles.add(Paths.get(destConfigFile));
+
+    // modify the configMape with cusotm logstash config file
+    String replaceStr = "msg";
+    logger.info("edit ConfigMap {0} to change 'message' to {1}", configMapName, replaceStr);
+    assertDoesNotThrow(() -> replaceStringInFile(destConfigFile, "message", replaceStr),
+        "replaceStringInFile failed");
+    assertTrue(replaceConfigMap(opNamespace, configMapName, destConfigFile),
+        String.format("Failed to replace configMap %s in namespace %s", configMapName, opNamespace));
+
+    verifyLogstashConfigMapModifyResult(replaceStr);
+  }
+
+  /**
+   * Test that users can config and create their own configMap weblogic-operator-logstash-cm
+   * when new variable createLogStashConfigMap sets to false.
```

Comment:
Remove the word new.
The description does not match the usecase implementation. Th test ends with a starting a second operator.  Does the test verify the custom configmap in the Operator namespace.

Reply:
means the users are able to helm install the Operator with their own configMap weblogic-operator-logstash-cm

Reply:
I am going to change the description to

   * Test that users are able to use their own configMap weblogic-operator-logstash-cm
   * to create the Operator when variable createLogStashConfigMap sets to false.

Reply:
removed and changed